example/app/models/Task.ts

```diff
@@ -0,0 +1,35 @@
+import {Realm, createRealmContext} from '@realm/react';
+export class Task extends Realm.Object {
+  _id!: Realm.BSON.ObjectId;
+  description!: string;
+  isComplete!: boolean;
+  createdAt!: Date;
+  userId!: string;
+
+  static generate(description: string, userId?: string) {
+    return {
+      _id: new Realm.BSON.ObjectId(),
+      description,
+      isComplete: false,
+      createdAt: new Date(),
+      userId: userId || '_SYNC_DISABLED_',
+    };
+  }
+
+  // To use a class as a Realm object type, define the object schema on the static property "schema".
+  static schema = {
+    name: 'Task',
+    primaryKey: '_id',
+    properties: {
+      _id: 'objectId',
+      description: 'string',
+      isComplete: {type: 'bool', default: false},
+      createdAt: 'date',
+      userId: 'string',
+    },
+  };
+}
+
+export default createRealmContext({
+  schema: [Task],
+});
```

Comment:
I think it's a bit confusing that the `Task.ts` is exporting this. I think we want to show a pattern that can be extended as new models are added. How about moving this to an `index.ts` and changing it to a named export called `TaskRealmContext` or something.

Reply:
oh yeah...that's a good idea.